 healthd: increase healthd fast timer to 10mins instead of 1min

healthd 1 min wakeups are causing significant power impact
during static display use case. so increasing the timer to 10min.

diff --git a/healthd/healthd.cpp b/healthd/healthd.cpp
index 28d6de7..ccbe0b0 100644
--- a/healthd/healthd.cpp
+++ b/healthd/healthd.cpp
@@ -41,7 +41,7 @@
  #ifdef QCOM_HARDWARE
   #define DEFAULT_PERIODIC_CHORES_INTERVAL_FAST (60 * 10)
  #else
-  #define DEFAULT_PERIODIC_CHORES_INTERVAL_FAST (60 * 1)
+  #define DEFAULT_PERIODIC_CHORES_INTERVAL_FAST (60 * 10)
  #endif
 #else
   #define DEFAULT_PERIODIC_CHORES_INTERVAL_FAST (BOARD_PERIODIC_CHORES_INTERVAL_FAST)
